pinot-common/src/main/java/org/apache/pinot/common/auth/AuthProviderUtils.java

```diff
@@ -63,6 +63,9 @@ public static AuthConfig extractAuthConfig(PinotConfiguration pinotConfig, Strin
    * @return auth provider
    */
   public static AuthProvider extractAuthProvider(PinotConfiguration pinotConfig, String namespace) {
```

Comment:
Do you need to put `@Nullable` annotation for `PinotConfiguration pinotConfig` and same for other methods?

Reply:
Didn't add it here because it is not added in other methods. In production code it should never be null.